docs/source/unit_catalog.rst

```diff
@@ -0,0 +1,32 @@
+Unit Catalog
+======================
+
+.. warning::
+    Unit Catalog is a new feature:
+      * The API specification is in beta.
+      * The SDK implementation is in alpha.
+
+    Thus, breaking changes may occur without further notice, see :ref:`appendix-alpha-beta-features` for more information.
```

Comment:
if breaking changes are expected, why this is not a part of 
https://github.com/cognitedata/cognite-sdk-python-experimental  instead?

Reply:
The experimental SDK is more or less deprecated